### PR TITLE
Restrict .p-inline-images items width

### LINKED ISSUE #724: Inline-images pattern displays images at their natural size
## Is this a proposal or an amendment?

- Amendment

Inline-images

## Visual

Current:
ubuntu in the cloud cloud ubuntu

Desired:
ubuntu in the cloud cloud ubuntu

## Context

The inline-images pattern does not limit the size of the logos. It should size them to fit nicely together.

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

Restrict width of images in `.p-inline-images`

## QA

- Pull and sync code
- Create `demo.html`
- Test using the following markup:

```html
<html>
<head>
  <title></title>
  <link rel="stylesheet" type="text/css" media="screen" href="build/css/build.css">
</head>
<body>

  <div class="p-strip">
    <div class="row">
        <h2>Inline images</h2>
        <div class="p-inline-images">
            <img class="p-inline-images__img" src="https://assets.ubuntu.com/v1/1ccb6d93-logo-deutschetelekom.svg" alt="Placeholder image" />
            <img class="p-inline-images__img" src="https://assets.ubuntu.com/v1/be345c7f-bSkyb_logo.png" alt="Placeholder image" />
            <img class="p-inline-images__img" src="https://assets.ubuntu.com/v1/595a3e03-lexisnexis.png" alt="Placeholder image" />
            <img class="p-inline-images__img" src="https://assets.ubuntu.com/v1/10e3a1bc-time-warner-cable.svg" alt="Placeholder image" />
            <img class="p-inline-images__img" src="https://assets.ubuntu.com/v1/379ae6d1-Yahoo_Japan_logo.png" alt="Placeholder image" />
            <img class="p-inline-images__img" src="https://assets.ubuntu.com/v1/b7693339-logo-bloomberg.svg" alt="Placeholder image" />
            <img class="p-inline-images__img" src="https://assets.ubuntu.com/v1/192f9755-partners-logo-nec-tagline.png" alt="Placeholder image" />
            <img class="p-inline-images__img" src="https://assets.ubuntu.com/v1/377e12ed-logo-ntt-communications.png" alt="Placeholder image" />
            <img class="p-inline-images__img" src="https://assets.ubuntu.com/v1/5482b658-liveperson-logo.jpg" alt="Placeholder image" />
            <img class="p-inline-images__img" src="https://assets.ubuntu.com/v1/4d6054f9-logo-cisco.svg" alt="Placeholder image" />
            <img class="p-inline-images__img" src="https://assets.ubuntu.com/v1/e71bd588-logo-samsung.svg" alt="Placeholder image" />
            <img class="p-inline-images__img" src="https://assets.ubuntu.com/v1/7ffe020f-logo-ebay.svg" alt="Placeholder image" />
            <img class="p-inline-images__img" src="https://assets.ubuntu.com/v1/b1fecbf0-logo-att-horizontal.png" alt="Placeholder image" />
            <img class="p-inline-images__img" src="https://assets.ubuntu.com/v1/b3315d88-walmart.svg" alt="Placeholder image" />    
        </div>
    </div>
  </div>

</body>
</html>
```

## Details

Fixes #724

### CODE REVIEW

**Comment by anthonydillon** [comment]
Here is an example of live:
ubuntu in the cloud cloud ubuntu

This QA:
demo html

**Comment by barrymcgee** [comment]
@anthonydillon It's impossible to restrict both width and height without distorting images - I can increase padding around each item and reduce number of items from 5 to 4 which will bring it closer to live.

**Comment by anthonydillon** [comment]
Small screens live:
ubuntu in the cloud cloud ubuntu

This QA:
demo html

File: scss/_patterns_inline-images.scss [hunk 1]
```diff
@@ -11,6 +11,15 @@
       display: inline-block;
       padding: .5rem;
       text-align: center;
+      width: 100%;
+
+      @media (min-width: $breakpoint-small) { // 2 items per row
+        max-width: calc(50% - 1rem);
```
Comment (anthonydillon) [inline scss/_patterns_inline-images.scss]: Can you make this the default styling so if takes effect on small screens.

File: scss/_patterns_inline-images.scss [hunk 2]
```diff
@@ -11,6 +11,15 @@
...
+      }
+
+      @media (min-width: $breakpoint-large) { // 5 items per row
+        max-width: calc(20% - 1rem);
```
Comment (anthonydillon) [inline scss/_patterns_inline-images.scss]: I believe we want 4 item per row from examples around our sites.

**Comment by anthonydillon** [comment]
@barrymcgee understand. I guess we dont get that sort of control without placing the images in a list.

**Comment by barrymcgee** [comment]
@anthonydillon Ready for re-review

File: scss/_patterns_inline-images.scss [hunk 3]
```diff
@@ -9,8 +9,18 @@
...
+      max-width: calc(50% - 2rem);
+      padding: 1rem;
       text-align: center;
+      width: 100%;
+
+      @media (min-width: $breakpoint-small) { // 2 items per row
+        max-width: calc(33.3333% - 2rem);
+      }
+
+      @media (min-width: $breakpoint-large) { // 5 items per row
```
Comment (anthonydillon) [inline scss/_patterns_inline-images.scss]: Please update comment to 4 items.

File: scss/_patterns_inline-images.scss [hunk 4]
```diff
@@ -9,8 +9,18 @@
 
     &__img {
       display: inline-block;
-      padding: .5rem;
+      max-width: calc(50% - 2rem);
+      padding: 1rem;
       text-align: center;
+      width: 100%;
+
+      @media (min-width: $breakpoint-small) { // 2 items per row
```
Comment (anthonydillon) [inline scss/_patterns_inline-images.scss]: Please update comment to 3 items.

File: scss/_patterns_inline-images.scss [hunk 5]
```diff
@@ -9,8 +9,18 @@
 
     &__img {
       display: inline-block;
-      padding: .5rem;
+      max-width: calc(50% - 2rem);
```
Comment (anthonydillon) [inline scss/_patterns_inline-images.scss]: Can you please add a comment of `// 2 items per row`

**Pull request merged**